packages/scaffold-utils/src/ConstantsUtil.ts

```diff
@@ -13,5 +11,5 @@ export const ConstantsUtil = {
   ADD_CHAIN_METHOD: 'wallet_addEthereumChain',
   EIP6963_ANNOUNCE_EVENT: 'eip6963:announceProvider',
   EIP6963_REQUEST_EVENT: 'eip6963:requestProvider',
-  VERSION: scaffoldUtilsPackageJson.version
+  VERSION: '4.0.2'
```

Comment:
Should this be `4.0.3`? @xzilja

Reply:
@svenvoskamp did we merge the auto-updater ???

Reply:
4.0.3 indeed. @tomiir I reverted auto updating as it was based on `assert {as json}` bit. It'll fail in quiet a few bundlers that have not implemented the standard yet.